jaeger_client/constants.py

```diff
@@ -33,10 +33,10 @@
 DEFAULT_FLUSH_INTERVAL = 1
 
 # Name of the HTTP header used to encode trace ID
-TRACE_ID_HEADER = b'uber-trace-id'
```

Comment:
I remember we explicitly ran into an issue with this string being Unicode in some instrumentation of urllib2. Why can we not keep this as `b`?

Reply:
Trying to fix this in #109